reports/postgres/repository.go

```diff
@@ -241,7 +242,17 @@ func (repo *PostgresRepository) ListReportsConfig(ctx context.Context, pm report
 		pgData += " OFFSET :offset"
 	}
 	pq := pageReportQuery(pm)
-	q := fmt.Sprintf(listReportsQuery, pq, pgData)
+
+	orderClause := ""
+	switch pm.Order {
```

Comment:
Let's try ordering by `COALESCE(updated_at, created_at)` in case `pm.Order` is empty.